python/ray/serve/master.py

```diff
@@ -407,7 +407,6 @@ def _scale_replicas(self, backend_tag, num_replicas):
                 del self.workers[backend_tag][replica_tag]
                 if len(self.workers[backend_tag]) == 0:
                     del self.workers[backend_tag]
-
```

Comment:
Accidental?

Reply:
Yeah, unless it was my linter. I'll try swapping it back.

Reply:
Yes, might have been linter. I removed it.